<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QWidget" name="AdminWindow">
  <property name="windowTitle"><string>管理员操作界面</string></property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QPushButton" name="btnViewStatistics">
     <property name="text"><string>查看统计</string></property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnQueryAccount">
     <property name="text"><string>查询账户</string></property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnFreezeAccount">
     <property name="text"><string>冻结账户</string></property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSetInterestRate">
     <property name="text"><string>设置利率</string></property>
    </widget>
   </item>
  </layout>
 </widget>
</ui>
